### PR TITLE
Add Braker3 

### PR DESCRIPTION
**Pull request by rlibouba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/braker3/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+categories: [Genome annotation]
+description: BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .
+homepage_url: https://github.com/Gaius-Augustus/BRAKER
+long_description: BRAKER is a pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes .
+owner: genouest
+name: braker
```
Comment (abretaud) [inline tools/braker3/.shed.yml:6]: braker -> braker3 (at all places)

File: tools/braker3/braker3.xml [hunk 2]
```diff
@@ -0,0 +1,168 @@
+<tool id="braker" name="Braker genome annotation" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01">
+    <description>version 3</description>
```
Comment (abretaud) [inline tools/braker3/braker3.xml:2]:
```suggestion
<tool id="braker3" name="BRAKER3" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="22.05">
    <description>genome annotation</description>
```

File: tools/braker3/macros.xml [hunk 3]
```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<macros>
+    <token name="@TOOL_VERSION@">2.1.6</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+
+    <xml name="requirements">
+        <requirement type="package" version="@TOOL_VERSION@">braker2</requirement>
+    </xml>
```
Comment (abretaud) [inline tools/braker3/macros.xml:8]: that's not up-to-date

**Review by abretaud — COMMENTED** [review]
First comments :)
Have your already updated the options, or is it just a copy of braker2 tool?

**Comment by rlibouba** [comment]
Hello, two out of three tests fail because the "proteins.fa" file seems to be too small. I tested locally with a much heavier dataset, and all tests passed. However, this dataset is too heavy to be put on github. Moreover, testing takes a long time with this dataset.

**Pull request merged**